bitnami/elasticsearch/README.md

```diff
@@ -391,21 +398,6 @@ As an alternative, this chart supports using an initContainer to change the owne
 
 You can enable this initContainer by setting `volumePermissions.enabled` to `true`.
 
-## Troubleshooting
```

Comment:
this section is useful for users no? that is a very common error

Reply:
Yes, I moved it inside the Configuration section, take a look some lines above